supabase/migrations/20240213092014_create_employee-table.sql

```diff
@@ -0,0 +1,67 @@
+create table "public"."Employee" (
+    "id" uuid not null,
+    "created_at" timestamp with time zone not null default now(),
+    "first_name" character varying not null,
+    "last_name" character varying not null,
+    "email" character varying not null,
+    "role" text[] not null
+);
+
+
+alter table "public"."Employee" enable row level security;
+
+CREATE UNIQUE INDEX "Employee_email_key" ON public."Employee" USING btree (email);
+
+CREATE UNIQUE INDEX "Employee_pkey" ON public."Employee" USING btree (id);
+
+alter table "public"."Employee" add constraint "Employee_pkey" PRIMARY KEY using index "Employee_pkey";
+
+alter table "public"."Employee" add constraint "Employee_email_key" UNIQUE using index "Employee_email_key";
+
+alter table "public"."Employee" add constraint "Employee_id_fkey" FOREIGN KEY (id) REFERENCES auth.users(id) ON DELETE CASCADE not valid;
+
+alter table "public"."Employee" validate constraint "Employee_id_fkey";
+
+grant delete on table "public"."Employee" to "anon";
```

Comment:
This is okay as is, but in I think it would be easier to read if you combine the grants into fewer commands.   For example,

# common grants
grant select, insert, update, delete on table blah to "anon"

# uncommon grants
grant references, trigger, truncate on table blah to "anon"

Reply:
Sounds good this was auto-generated creating migrations, I will make the change accordingly.